crates/types/src/data.rs

```diff
@@ -692,4 +698,29 @@ pub mod null_block {
             Err(_) => None,
         }
     }
+
+    /// Builder fee data for a null block payload
+    #[must_use]
+    pub fn fee_data<TYPES: NodeType>(num_storage_nodes: usize) -> Option<FeeData<TYPES>> {
+        let (_, priv_key) =
+            <TYPES::BuilderSignatureKey as BuilderSignatureKey>::generated_from_seed_indexed(
+                [0_u8; 32], 0,
+            );
+
+        let (null_block, null_block_metadata) =
+            <TYPES::BlockPayload as BlockPayload>::from_transactions([]).ok()?;
+
+        match TYPES::BuilderSignatureKey::sign_fee(
+            &priv_key,
+            0,
```

Comment:
Could this either not be a magic number or be something like `0 /* what this value is */`?